src/expander_pipeline.jl

```diff
@@ -961,12 +961,13 @@ function create_docsnode(docstrings, results, object, page, doc)
     slug = Documenter.slugify(object)
     anchor = Documenter.anchor_add!(doc.internal.docs, object, slug, page.build)
     docsnode = DocsNode(anchor, object, page)
-    # Convert docstring to MarkdownAST, convert Heading elements, and push to DocsNode
-    for (markdown, result) in zip(docstrings, results)
-        ast = convert(Node, markdown)
-        doc.user.highlightsig && highlightsig!(ast)
-        # The following 'for' corresponds to the old dropheaders() function
+    function recursive_heading_to_bold!(ast)
```

Comment:
```suggestion
    function recursive_heading_to_bold!(ast::MarkdownAST.Node)
```

Reply:
Maybe let's pull this into a separate function, not a closure. It doesn't depend on any of the variables from `create_docsnode`, right?

Reply:
done